Vagrantfile

```diff
@@ -24,6 +24,7 @@ Vagrant.require_version '>= 2.2.5'
   s2_search:    { ip: '192.168.56.36', hostname: 's2.search.splunk', ports: { 8017 => 8000, 8107 => 8089 } },
   s2_slave1:    { ip: '192.168.56.37', hostname: 's2.slave01.splunk', ports: { 8018 => 8000, 8108 => 8089 } },
   s2_slave2:    { ip: '192.168.56.38', hostname: 's2.slave02.splunk', ports: { 8019 => 8000, 8109 => 8089 } },
+  sf_standalone:{ ip: '192.168.56.39', hostname: 'splunk3', ports: { 8020 => 8000, 8110 => 8089 } },
```

Comment:
it doesn't look like this is used, so it can be deleted.

Reply:
Removed: https://github.com/cerner/cerner_splunk/pull/265/commits/3c791c8b0b39737525dc63252bed91702aa74c01